lib/puppet/application/cert.rb

```diff
@@ -100,219 +99,19 @@ def subcommand=(name)
-  needs to be restarted after revoking certificates.
-
-* sign:
-  Sign an outstanding certificate request. If '--interactive' or '-i' is
-  supplied the user will be prompted to confirm that they are signing the
-  correct certificate (recommended). If '--assume-yes' or '-y' is supplied
-  the interactive prompt will assume the answer of 'yes'.
-
-* verify:
-  Verify the named certificate against the local CA certificate.
-
-* reinventory:
-  Build an inventory of the issued certificates. This will destroy the current
-  inventory file specified by 'cert_inventory' and recreate it from the
-  certificates found in the 'certdir'. Ensure the puppet master is stopped
-  before running this action.
-
-OPTIONS
--------
-Note that any setting that's valid in the configuration
-file is also a valid long argument. For example, 'ssldir' is a valid
-setting, so you can specify '--ssldir <directory>' as an
-argument.
-
-See the configuration file documentation at
-https://puppet.com/docs/puppet/latest/configuration.html for the
-full list of acceptable parameters. A commented list of all
-configuration options can also be generated by running puppet cert with
-'--genconfig'.
-
-* --all:
-  Operate on all items. Currently only makes sense with the 'sign',
-  'list', and 'fingerprint' actions.
-
-* --allow-dns-alt-names:
-  Sign a certificate request even if it contains one or more alternate DNS
-  names. If this option isn't specified, 'puppet cert sign' will ignore any
-  requests that contain alternate names.
-
-  In general, ONLY certs intended for a Puppet master server should include
-  alternate DNS names, since Puppet agent relies on those names for identifying
-  its rightful server.
-
-  You can make Puppet agent request a certificate with alternate names by
-  setting 'dns_alt_names' in puppet.conf or specifying '--dns_alt_names' on the
-  command line. The output of 'puppet cert list' shows any requested alt names
-  for pending certificate requests.
-
-* --allow-authorization-extensions:
-  Enable the signing of a request with authorization extensions. Such requests
-  are sensitive because they can be used to write access rules in Puppet Server.
-  Currently, this is the only means by which such requests can be signed.
-
-* --digest:
-  Set the digest for fingerprinting (defaults to the digest used when
-  signing the cert). Valid values depends on your openssl and openssl ruby
-  extension version.
-
-* --debug:
-  Enable full debugging.
-
-* --help:
-  Print this help message
-
-* --verbose:
-  Enable verbosity.
-
-* --version:
-  Print the puppet version number and exit.
-
-
-EXAMPLE
--------
-    $ puppet cert list
-    culain.madstop.com
-    $ puppet cert sign culain.madstop.com
-
-
-AUTHOR
-------
-Luke Kanies
-
-
-COPYRIGHT
----------
-Copyright (c) 2011 Puppet Inc., LLC Licensed under the Apache 2.0 License
-
-    HELP
+    _("This command is no longer functional, please use `puppetserver ca` instead.")
```

Comment:
It'd be nice to update this to exactly what folks can use instead.

Reply:
Like per command?